browser/browsing_data/brave_clear_browsing_data.cc

```diff
@@ -0,0 +1,188 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/browsing_data/brave_clear_browsing_data.h"
+
+#include <vector>
+
+#include "base/run_loop.h"
+#include "base/scoped_observer.h"
+#include "base/trace_event/common/trace_event_common.h"
+#include "chrome/browser/browser_process.h"
+#include "chrome/browser/browsing_data/chrome_browsing_data_remover_delegate.h"
+#include "chrome/browser/lifetime/browser_shutdown.h"
+#include "chrome/browser/profiles/profile.h"
+#include "chrome/browser/profiles/profile_manager.h"
+#include "chrome/common/pref_names.h"
+#include "components/browsing_data/core/pref_names.h"
```

Comment:
you need to a dep for this here https://github.com/brave/brave-core/pull/1213/files#diff-84f8700538180eec90bb2f4c9dff4da4R79

Reply:
Added.